packages/api-headless-cms/__tests__/multipleValues.test.js

```diff
@@ -121,4 +121,31 @@ describe("Multiple Values Test", () => {
             `Cannot change "multipleValues" for the "tags" field because it's already in use in created content.`
         );
     });
+
+    it(`should not be able to set "multipleValues" as index`, async () => {
+        const { createContentModel, updateContentModel, content } = environment(ids.environment);
+
+        let error;
+        const productContentModel = await createContentModel(
+            mocks.blogWithTagsSetToMultipleValue({ contentModelGroupId: ids.contentModelGroup })
```

Comment:
While the mock you've chosen here works, let's copy it into a separate file. Notice that basically each test file has a corresponding mock file in the `mocks` folder. Let's keep it that way.

This helps us to avoid breaking tests if adjustments on the mocks are needed (e.g. me breaking your test because I had to change something on it, and vice versa).

Reply:
Other than that, the test looks good. 👍

Reply:
See new changes to PR